config/sets/jms/annotations-to-attributes.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Php80\Rector\Class_\AnnotationToAttributeRector;
+use Rector\Php80\ValueObject\AnnotationToAttribute;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+    $services->set(AnnotationToAttributeRector::class)
+        ->configure([
+            new AnnotationToAttribute('JMS\Serializer\Annotation\Accessor'),
+            new AnnotationToAttribute('JMS\Serializer\Annotation\AccessorOrder'),
+            new AnnotationToAttribute('JMS\Serializer\Annotation\AccessType'),
```

Comment:
Could you link the responsible PR in jms repository? Just for future context

Reply:
There are multiple ones:
 * https://github.com/schmittjoh/serializer/pull/1320
 * https://github.com/schmittjoh/serializer/pull/1332
 * https://github.com/schmittjoh/serializer/pull/1337

Reply:
I mean in the set as a comment. It is very helpful to have them available in case of edge cases